source/docs/articles/drupal/8.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
@bmackinney - can you review the landing page again? Revised the previous commit to include suggestions from slack discussion.